<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListWindow</class>
 <widget class="QMainWindow" name="ListWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow#ListWindow
{
	background-color: rgb(38, 44, 56);
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
	background-color: rgba(67, 67, 67,0.9);
color: rgb(255, 255, 255);
font: 14pt &quot;华光行楷_CNKI&quot;;
border:2px solid;
border-bottom-left-radius:1em;
border-bottom-right-radius:1em;
border-top-left-radius:1em;
border-top-right-radius:1em;
border-style:solid;
border-width:3px;
}
QPushButton#Order_Bt:pressed
{
	background-color: rgb(170, 170, 0);
}
QPushButton#finishBt:pressed

{
	background-color: rgb(170, 170, 0);
}
QLabel#AllLB
{
	
	color: rgb(255, 255, 255);
}
QLabel#lable_11
{
	
	color: rgb(255, 255, 255);
}
QLabel#lable_12
{
	
	color: rgb(255, 255, 255);
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,4">
    <property name="spacing">
     <number>1</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="6,1,1">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,1,2,1,2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lable_11">
          <property name="font">
           <font>
            <family>华文行楷</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>总金额：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="AllLB">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>15</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>华文隶书</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="lineWidth">
           <number>10</number>
          </property>
          <property name="text">
           <string>0</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="indent">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lable_12">
          <property name="font">
           <font>
            <family>华文隶书</family>
            <pointsize>15</pointsize>
           </font>
          </property>
          <property name="text">
           <string>元</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="finishBt">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>结账</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Order_Bt">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>下单</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="returnBt">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);
font: 14pt &quot;华光行楷_CNKI&quot;;
border:2px solid;
border-bottom-left-radius:1em;
border-bottom-right-radius:1em;
border-top-left-radius:1em;
border-top-right-radius:1em;
border-style:solid;
border-width:3px;
background-color: rgba(67, 67, 67,0.9)</string>
          </property>
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,6" rowminimumheight="0,0">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>华文隶书</family>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255,0.9);</string>
        </property>
        <property name="text">
         <string>         温馨提示：您点主食了吗？</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QTableWidget" name="tableWidget">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>600</height>
         </size>
        </property>
        <property name="cornerButtonEnabled">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>0</number>
        </property>
        <property name="columnCount">
         <number>0</number>
        </property>
        <attribute name="horizontalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>180</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>225</number>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>180</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>225</number>
        </attribute>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
